addons/website_livechat/__manifest__.py

```diff
@@ -25,6 +25,7 @@
         "im_livechat.assets_embed_core": [
             "website/static/src/**/common/**/*",
             "website_livechat/static/src/**/common/**/*",
+            "website_livechat/static/src/embed/shadow.scss",
```

Comment:
Those bundles are not following mail bundle guidelines. 
- `assets_embed_core` should include `website_livechat/static/src/embed/**/*`
- `assets_backend` shouldn't include everything, this would remove need to remove what's in the embed folder.